src/styles/header.scss

```diff
@@ -0,0 +1,42 @@
+@import 'variables';
```

Comment:
Ensure that the 'variables' file being imported contains definitions for `$nav-link-color` and `$nav-link-hover-color`. Missing variable definitions will result in styling issues.